Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "fuel-telemetry"
+version = "0.1.0"
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/fuel-telemetry"
+description = "A tracing library to implement Fuel telemetry"
+
+[lib]
+name = "fuel_telemetry"
+path = "src/lib.rs"
+
+[dependencies]
```

Comment:
nit: since this is a new repo; id reccommend minimal lib size for each dependency.
This can be done via setting `default-features = false` and explicitly pulling in features that the code needs.
This should also reduce binary size.
e.g.
```
reqwest = { version = "0.12", default-features = false, features = ["blocking"] }
```

Reply:
Yep, sounds good. Thanks!